docs/import/manage_disaster.js

```diff
@@ -675,41 +683,87 @@ function toggleState(known) {
   }
 }
 
+const scoreAssetTypes = [
+  ['poverty', ['snap_data', 'paths'], 'Poverty'],
```

Comment:
maybe do some magic to combine the 0 and 2 index here?

Reply:
What do you suggest? Putting them in a sub-array? Or putting them closer together in this array? I'd rather not make a big structured thing like {key: ..., path: [...], display_name: ...}

Reply:
Oh,  I meant since [2] seems to be [0] with a capitalized first letter maybe we can just store one and derive the other from it. Unless you think that pattern will change going forward?

Reply:
Yeah, I think it's not to be relied on.